developers/arbitrum-integration.md

```diff
@@ -0,0 +1,74 @@
+---
+description: An overview of the integration of Arbitrum Orbit and Nitro with Celestia, detailing the key features and benefits, including the Ethereum fallback mechanism.
+---
+
+# Introduction to Arbitrum rollups with Celestia as DA
```

Comment:
maybe a cool banner image here?

Reply:
great idea, how about this as first version? https://blog.celestia.org/content/images/size/w2000/2023/10/Celestia_Arbitrum2.jpg

Reply:
https://github.com/celestiaorg/docs/pull/1323/commits/307bb98851ac9a87ed624376151ab273e0406c29 https://github.com/celestiaorg/docs/pull/1323/commits/307bb98851ac9a87ed624376151ab273e0406c29

Reply:
![DALL·E 2023-12-13 17 25 19 - Create a sophisticated and professional graphic in a 16_9 aspect ratio, combining the themes of Arbitrum and Celestia for a technical documentation pa](https://github.com/celestiaorg/docs/assets/46639943/f2fe518f-8b0c-4922-9677-ed5b6c55ef2c)
it would also be fun to use something like this in future version (ai gen)